### PR TITLE
DEMO-77: As a user, I want to see the detail of the image

### PR DESCRIPTION
**Pull request by clash82**
https://jira.ez.no/browse/DEMO-77

nkqhkkzlsy

### CODE REVIEW

**Comment by clash82** [comment]
review please @SylvainGuittard

**Comment by clash82** [comment]
update: added thin bottom border for images

File: index.ios.js [hunk 1]
```diff
@@ -51,20 +53,25 @@ export default class PlacesAndTastes extends Component {
...
+            endpoint = "/api/ezp/v2/views",
+            body = `<?xml version="1.0" encoding="UTF-8"?>
+        
         <ViewInput>
             <identifier>AppImages</identifier>
             <Query>
                 <Criteria>
                     <ContentTypeIdentifierCriterion>image</ContentTypeIdentifierCriterion>
                     <ParentLocationIdCriterion>206</ParentLocationIdCriterion>
+                    <VisibilityCriterion>visible</VisibilityCriterion>
```
Comment (SylvainGuittard) [inline index.ios.js:74]: Very good. I forgot to commit this new `Criterion` last week.

**Review by SylvainGuittard — APPROVED** [review]

File: index.ios.js (hunk 1 above)
Comment (clash82) [reply index.ios.js:74]: Thank you :-)

**Pull request merged**